.github/workflows/publish-release.yml

```diff
@@ -62,6 +62,9 @@ jobs:
         uses: dtolnay/rust-toolchain@master
         with:
           toolchain: stable
+      - run: cargo publish -p oranda-generate-css --token ${CRATES_TOKEN}
+        env:
+          CRATES_TOKEN: ${{ secrets.CRATES_TOKEN }}
       - run: cargo publish --token ${CRATES_TOKEN}

```

Comment:
not 100% confident this will do the right thing, but it probably should...? (paranoid brain wants to add -p oranda here to be unambiguous)